syntax = "proto3";
package mongo;




import "google/protobuf/struct.proto";

/*
RemoveAll请求
*/
message RemoveAllRequest {
    /*
    表名
    */
    string collection = 1 ;
    /*
    删除条件
    */
    google.protobuf.Struct selector = 2 ;
}

/*
RemoveAll返回
*/
message RemoveAllResponse {
    /*
    删除总数
    */
    int32 removed = 1 ;
}

/*
RemoveAllApi返回
*/
message RemoveAllResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    mongo.RemoveAllResponse data = 4 ;
}

